src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,42 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  padding: 0;
+
+  &__box {
+    display: flex;
+
+    @for $index from 1 through 5 {
+      &--#{$index} :nth-child(-n + #{$index}) {
+        background-image: url(/images/star-active.svg);
+      }
+    }
+  }
+
+  &__star {
+    margin-right: 4px;
+    height: 16px;
+    width: 16px;
+    border-radius: 1px;
+
+    background-image: url(/images/star.svg);
+    background-repeat: no-repeat;
+    background-position: 50%;
+
+    &:first-child {
+      margin-left: 16px;
```

Comment:
do not set spacing this way for every element
<img width="224" alt="image" src="https://user-images.githubusercontent.com/66749919/196484942-d2108518-f21e-4c23-b186-ec6439f08fdc.png">

use paddings for the card like this
<img width="225" alt="image" src="https://user-images.githubusercontent.com/66749919/196484868-ad1abfa7-9d5d-4b96-9536-26a823750671.png">